test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -125,22 +125,22 @@ public void Build_honors_UseStartup_with_string()
         }
 
         [Fact]
-        public void DefaultConfigurationCapturesStartupErrors()
+        public void DefaultConfigurationDoesNotCapturesStartupErrors()
```

Comment:
Should be ...CaptureStartup... instead of ...CapturesStartup... right? Should it also be "DefaultHostingConfiguration..."?